axelrod/match.py

```diff
@@ -119,6 +119,6 @@ def play(self):
 
     def sparklines(self, c_symbol=u'█', d_symbol=u' '):
         return (
-            sparkline(self._player1.history, c_symbol, d_symbol) +
+            sparkline(self.player1.history, c_symbol, d_symbol) +
             u'\n' +
-            sparkline(self._player2.history, c_symbol, d_symbol))
+            sparkline(self.player2.history, c_symbol, d_symbol))
```

Comment:
My understanding is that the underscore indicates whether an attribute should be considered 'private' i.e. only accessed from within the object itself. I don't think it has anything to do with whether a user should be accessing it.